### PR TITLE
feat(strategy): deploy mainnet

### PR DESCRIPTION
**Pull request by gpsanant**
added mainnet strategies

### CODE REVIEW

File: script/output/strategy/strategy_deployment_mainnet_2023_12_5.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "chainInfo": {
+    "chainId": 1,
+    "deploymentBlock": 18721129
+  },
+  "strategies": {
+    "ETHx": "0x9d7eD45EE2E8FC5482fa2428f15C971e6369011d",
+    "ankrETH": "0x13760F50a9d7377e4F20CB8CF9e4c26586c658ff",
+    "oETH": "0xa4C637e0F704745D182e4D38cAb7E7485321d059",
```
Comment (wadealexc) [inline script/output/strategy/strategy_deployment_mainnet_2023_12_5.json:9]: oETH has a capital O - "OETH"

Comment (wadealexc) [reply script/output/strategy/strategy_deployment_mainnet_2023_12_5.json:9]: Need to fix this in configs as well

Comment (gpsanant) [reply script/output/strategy/strategy_deployment_mainnet_2023_12_5.json]: done

**Review by ChaoticWalrus — APPROVED** [review]
LGTM but presumably want to also add these contracts to the table in the main README

**Comment by gpsanant** [comment]
> LGTM but presumably want to also add these contracts to the table in the main README

just did, can you double check?

**Comment by ChaoticWalrus** [comment]
> just did, can you double check?

Thanks, looks good, going to merge this.

**Pull request merged**